packages/designmanual/stories/molecules/NotionListExample.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2021-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React from 'react';
+import styled from '@emotion/styled';
+import { breakpoints, mq } from '@ndla/core';
+import NotionBlock from '../molecules/NotionBlock';
+
+const NotionListTitleWrapper = styled.div`
+  margin: 48px 0 48px;
+  position: relative !important;
+  right: auto !important;
+  left: -16.6666666667%;
+  width: 133.3333333333% !important;
+  padding-left: 24px;
+  padding-right: 24px;
+  ${mq.range({ until: breakpoints.tabletWide })} {
+    width: 100%;
+    left: -4%;
+  }
+`;
+const NotionListTitle = styled.h2`
+  font-size: 24px;
+  color: #444444;
+
+  border-bottom: 1px solid #d1d6db;
+  padding-bottom: 24px;
+`;
+type Notions = {
+  type: 'image' | 'video' | 'H5P';
+};
+type Props = {
+  notionBlocks: Notions[];
+  title: string;
+};
+
+const NotionListExample = ({ title, notionBlocks }: Props) => {
+  return (
+    <>
+      {notionBlocks && (
+        <>
+          <NotionListTitleWrapper>
+            <NotionListTitle>{title}</NotionListTitle>
+          </NotionListTitleWrapper>
+          {notionBlocks.map((x) => (
+            <NotionBlock type={x.type}></NotionBlock>
+          ))}
+        </>
+      )}
+    </>
+  );
+};
```

Comment:
Det er dette som er visning av forklaringer i liste? Det hadde ikkje vore naturlig å lage dette som en komponent som tar inn en tittel som ReactNode, og en liste med forklaringer som kan vises som barn? Då slipper vi å gjenskape denne funksjonaliten både i ed og i article-converter. Denne koden er veldig knytta til kode i designmanualen og ikkje noko som kan gjenbrukes direkte.

Reply:
Jeg lagde en gjenbrukbar komponent her: https://github.com/NDLANO/frontend-packages/tree/search-ungrouped/packages/ndla-ui/src/Notion. `ConceptNotion` fungerer som en drop-in erstatning for komponentene som vises fram i designmanual-eksempler.

Reply:
Eg fant ikkje en komponent under lenka du la inn her. Er det noko som ikkje er pusha endå?

Reply:
https://github.com/NDLANO/frontend-packages/blob/search-ungrouped/packages/ndla-ui/src/Notion/ConceptNotion.tsx

Reply:
Ja, dette er en forklaring, men det hadde vore lurt med en komponent for å vise ei liste med forklaringer. Ref https://github.com/NDLANO/Issues/issues/3015